spec/uploader_spec.rb

```diff
@@ -385,6 +385,21 @@
     it "should fail if timeout is reached" do
       expect{Cloudinary::Uploader.upload(Pathname.new(TEST_IMG), :tags => [TEST_TAG, TIMESTAMP_TAG])}.to raise_error(RestClient::RequestTimeout)
     end
+
+    it "should allow passing nil value" do
```

Comment:
This assumes a certain initial state of `Cloudinary.config`, maybe want to make that explicit?

Reply:
We first check immediate options and if its not there, fall back to config, and config is set to something not `nil` on line 379